src/App.js

```diff
@@ -1,27 +1,74 @@
 import React, { Component } from 'react';
 import './App.css';
 import GroupList from './GroupList';
+import TodoList from './TodoList';
 
 class App extends Component {
   state = {
-    groupList: [{ name: 'sally' }, { name: 'sun' }],
+    groupList: [
+      {
+        name: 'coding',
+        todoList: [
+          { content: 'todo app', completed: false },
+          { content: 'next.js', completed: false },
+        ],
+      },
+    ],
     selectedIndex: null,
   };
 
   addGroup = (name) => {
+    const { groupList } = this.state;
+
+    this.setState({
+      groupList: [...groupList, { name, todoList: [] }],
+      selectedIndex: groupList.length,
+    });
+  };
+
+  selectGroup = (index) => {
+    this.setState({ selectedIndex: index });
+  };
```

Comment:
```suggestion
  selectGroup = (selectedIndex) => {
    this.setState({ selectedIndex });
  };
```
변수 이름을 selectedIndex로 하면 코드가 조금 더 간결해질것 같아요.

Reply:
그렇게 하면 30번 줄이 더 간결해 질 수 있겠네요~
그렇지만 제 생각에는 selectGroup이란 메소드가 인자로 selectedIndex를 받으면
의미가 헷갈릴 수도 있겠다는 생각이 듭니다.

- selectGroup(index)               : 선택한 그룹의 인덱스를 받는다.
- selectGroup(selectedIndex) : 선택한 그룹의 선택된 인덱스를 받는다.

선택한 그룹의 선택된 인덱스는 뭐지?? 
이렇게 헷갈릴 수도 있을 것 같아서 30번 줄은 기존 형식을 유지하겠습니다.

객체의 키 값과 변수의 이름이 같을 때 키 값이 생략 가능하다는 문법은 필요 시 적절히 활용하겠습니다.
감사합니다. :)